apps/client/src/app/pages/search/search/search.component.ts

```diff
@@ -755,4 +760,18 @@ export class SearchComponent implements OnInit {
   trackByItem(index: number, item: SearchResult): number {
     return +item.itemId;
   }
+
+  public adjust(form: string, prop: string, amount: number, min: number, max: number, arrayName?: string, arrayIndex?: number): void {
```

Comment:
```suggestion
  public adjust(form: keyof SearchComponent, prop: string, amount: number, min: number, max: number, arrayName?: string, arrayIndex?: number): void {
```

This small change makes sure you're giving a form name that exists.

Reply:
You mean prop: keyof SerchComponent no ?

Reply:
no, form, because form is a member of SearchComponent

Reply:
Or not, but SearchComponent is not the good type, with FormGroup it work's, don't know if it's what you think about

Reply:
the form you're passing is a name, the name of the form inside of the SearchComponent component, which is why it's this type.

For instance, one of the values used is `instanceFiltersForm`, which is a property of SearchComponent.